### DISCUSSION TITLE
Transition delay between multiple playlists.

### DISCUSSION THREAD

**Discussion by ugacomp** [ant-media/Ant-Media-Server#6797] [FAQs]
Hello! 
I noticed that there is a transition delay between video files in the playlist. 2 seconds are skipped when the new video plays. Also there is a transition delay. 

I am running Community Edition 2.10.0 20240705_1023

@yashtandon113 
@mekya 
@muratugureminoglu 
@asharamseervi 
@Mohit-3196

**Comment by asharamseervi** [comment]
Hi @ugacompIT, I'm not sure what could be causing that 2-second skip.

I recommend trying the latest community edition in a different environment and testing the same source to see if it works better with the new environment or if the issue still exists.

Probably, sharing log or supporting through call may help further identifying the glitch if that still exists.

**Comment by yashtandon113** [comment]
Hi @ugacompIT 

Check out the document below to optimize your playlist playback experience.

https://antmedia.io/docs/guides/publish-live-stream/playlist/creating-playlist/#optimizing-playlist-for-better-playback-experience

I hope it helps.
